internal/testpb/generate.sh

```diff
@@ -2,6 +2,8 @@
 # generate .pb.go file from .proto file.
 set -e
 protoc --go_out=plugins=grpc:. test.proto
-echo '//go:generate ./generate.sh
+echo '
+//go:generate ./generate.sh
 ' >> test.pb.go
+goimports -w test.pb.go
```

Comment:
Is it possible to include other go files too (if added later)?

Reply:
It is possible to make it generic but i would like to defer that until we have another proto file.

Reply:
SG